src/nexmo/__init__.py

```diff
@@ -141,7 +145,7 @@ def send_message(self, params):
             })
         :param dict params: A dict of values described at `Send an SMS <https://developer.nexmo.com/api/sms#send-an-sms>`_
         """
-        return self.post(self.host, "/sms/json", params, supports_signature_auth=True)
+        return self.post(self.host, "/sms/json", params, signature_auth=True)
```

Comment:
I think this change is wrong - pretty sure in a previous review we said supports_signature_auth was correct - were you working off an old baseline?